plugins/vppv2/l3plugin/descriptor/static_route.go

```diff
@@ -89,7 +92,7 @@ func (d *RouteDescriptor) EquivalentRoutes(key string, oldRoute, newRoute *l3.St
 		oldRoute.GetVrfId() != newRoute.GetVrfId() ||
 		oldRoute.GetViaVrfId() != newRoute.GetViaVrfId() ||
 		oldRoute.GetOutgoingInterface() != newRoute.GetOutgoingInterface() ||
-		oldRoute.GetWeight() != newRoute.GetWeight() ||
+		getWeight(oldRoute) != getWeight(newRoute) ||
```

Comment:
Why doesn't this compare default value as `0` and then in vppcalls package set `1` in the binapi request instead of `0`.

Reply:
No this fixes comparison between what is dumped and what is defined by NB,
The unspecified fields may remain empty in the binapi calls, but dump will return them set to the default values and resync would trigger modify if we do not consider them in the comparison function.